docs/smart_rollups.md

```diff
@@ -7,7 +7,7 @@ author: Davis Sawali
 
 Rollups are a permissionless scaling implementation for the Tezos blockchain. The idea is that anyone can originate and operate one or more rollups, increasing the overall throughput of the Tezos blockchain.
 
-In Taquito, we have implemented some of the operations included in Mumbai protocol update in regards to smart rollups. In this document, we will go through the operations we support. We also won't go too detailed on how rollups work behind the scenes, if you'd like to understand the feature a bit deeper, you can refer to [this document](https://tezos.gitlab.io/mumbai/smart_rollups.html).
+In Taquito, we have implemented some of the operations included in Mumbai protocol update in regards to smart rollups. In this document, we will go through the operations we support. We also won't go too detailed on how rollups work behind the scenes, if you'd like to understand the feature a bit deeper, you can refer to [this document](https://tezos.gitlab.io/oxford/smart_rollups.html).
```

Comment:
paragraph still mentions Mumbai, we should probably rephrase it in a way where we clearly explain that the smart rollup changes were initially rolled out in the Mumbai update but the feature still lives in current protocols

Reply:
great idea, will rephrase the sentence as 

`In Taquito, we have implemented some of the smart rollups operations which are first introduced in Mumbai protocol.`